sdk-extension/opentelemetry-sdk-extension-aws/src/opentelemetry/sdk/extension/aws/resource/ecs.py

```diff
@@ -51,6 +51,7 @@ def detect(self) -> "Resource":
                         line = raw_line.strip()
                         if len(line) > _CONTAINER_ID_LENGTH:
                             container_id = line[-_CONTAINER_ID_LENGTH:]
+                            break
```

Comment:
is it possible to add a unit test for this?

Reply:
I modified the existing unit tests which had the container ID multiple times to use `bogusContainerIdThatShouldNotBeOneSetBecauseTheFirstOneWasPicked` on subsequent lines. That way if the code is missing this break statement it will pick up the wrong container ID because it didn't grab the first one 😄